wdl-lint/tests/lints/runtime-keys-wdl-1.2/source.wdl

```diff
@@ -1,12 +1,14 @@
-#@ except: BlankLinesBetweenElements, CommentWhitespace, DescriptionMissing
-#@ except: MissingRequirements
+## TODO: A strange one: this emits two errors, both from MissingRequiremnts.
```

Comment:
It seems like to me the two new diagnostics are valid, just being excepted prior; the test simply didn't want it complaining about adding `container` to the `runtime` section when we're also telling them to replace it with a requirements section.

Reply:
IMO
> warning[MissingRequirements]: task `a_task_with_no_keys` is missing a `requirements` section

is a misfire. Technically, it is "missing" a requirements section, **but** it has a `runtime` section which is "good enough" for actually running the task. We should only be emitting the deprecated `runtime` diagnostic.